model-optimizer/extensions/front/onnx/faster_rcnn.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "custom_attributes":
```

Comment:
Please, update the `docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_Mask_RCNN.md` or create a separate for the Faster-RCNN. Do not forget to update the `docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md` and `docs/doxygen/ie_docs.xml`